xclim/core/indicator.py

```diff
@@ -310,15 +315,15 @@ def missing(*args, **kwds):
         freq = kwds.get("freq")
         indexer = kwds.get("indexer") or {}
 
-        # We flag any period with missing data
-        miss = (checks.missing_any(da, freq, **indexer) for da in args)
+        # We flag periods according to the currently set missing data method
+        miss = (checks.missing_from_context(da, freq, **indexer) for da in args)
```

Comment:
I think this can be the default, but I suggest Indicators support the possibility of imposing a given missing function that could not be modified by the context. We can probably do this in the Indicator refactoring however.

Reply:
That's a good idea. Then we could move what is currently in the `Indicator.missing` method to the `__call__` and set `Indicator.missing` to  `checks.missing_from_context`.

Reply:
Or even, scratch `missing_from_context` and put this mechanism in the base `Indicator.missing`. Is there really a need for  `mìssing_from_context` outside of the Indicators? I guess a user would make the explicit choice of method and options...

Reply:
I don't see one, so yes, `missing_from_context` could be a method of `Indicator`. If `missing` is not explicitly passed to Indicator, then it's using the default. I suggest we make this change in the refactoring branch however.